tests/adapters/compliance_tests/add_remove_interface_to_bond.py

```diff
@@ -0,0 +1,25 @@
+from hamcrest import is_, assert_that, has_item
+from netman.core.objects.exceptions import InterfaceNotInBond
+from tests.adapters.compliance_test_case import ComplianceTestCase
+
+
+class AddRemoveInterfaceToBond(ComplianceTestCase):
```

Comment:
One compliance test per method, the name of the test should be MethodNameTest

Sorry for being pedantic.
This is aimed to auto generate a supported method/driver doc from the test results